src/components/forms/PlanForm/helpers.ts

```diff
@@ -769,7 +773,7 @@ export function getPlanFormValues(planObject: PlanDefinition): PlanFormFields {
     taskGenerationStatusContext.length > 0
       ? (taskGenerationStatusContext[0].valueCodableConcept as taskGenerationStatusType)
       : isDynamicPlan(planObject)
-      ? taskGenerationStatuses[2]
+      ? taskGenerationStatuses[4]
       : taskGenerationStatuses[1];
```

Comment:
Lets use a map instead of these indices.  It's easier to read that way.

Reply:
+ 1 on this